src/foam/nanos/pm/services.jrl

```diff
@@ -0,0 +1,7 @@
+p({"class":"foam.nanos.boot.NSpec", "name":"pmLogger",                         "lazy":true,  "serve":false, "serviceClass":"foam.nanos.pm.DAOPMLogger"})
+p({"class":"foam.nanos.boot.NSpec", "name":"pmInfoDAO",                        "lazy":true,  "serve":true,  "serviceScript":"return new foam.dao.ProxyDAO.Builder(x).setDelegate(new foam.dao.MDAO(foam.nanos.pm.PMInfo.getOwnClassInfo())).build();","client":"{\"of\":\"foam.nanos.pm.PMInfo\", \"remoteListenerSupport\": true}" })
```

Comment:
Why do all of these services have a plain `ProxyDAO` decorating them?

Reply:
I really only need the plain ProxyDAO on the pmInfoDAO. I use it to replace the delegate during rollup.  After that it was cut and paste.